jetstream/analysis.py

```diff
@@ -412,13 +412,15 @@ def validate(self) -> None:
             """WITH enrollments_table AS (
                 SELECT '00000' AS client_id,
                     'test' AS branch,
+                    '1' AS message_id,
```

Comment:
this is to make validation check pass for spotlight outcome

Reply:
This should not be necessary here. The outcome definition needs to be updated to select `message_id` in the data source definition: https://github.com/mozilla/jetstream-config/pull/185/files/965d27c20585bd1f53da8ffbb47741d1bbcff362..45c539db7867c91817a0cb5817639c1e5c0e50b8#r891746846
```suggestion
```

Reply:
This should not be necessary here. The outcome definition needs to be updated to select `message_id` in the data source definition: https://github.com/mozilla/jetstream-config/pull/185/files/965d27c20585bd1f53da8ffbb47741d1bbcff362..45c539db7867c91817a0cb5817639c1e5c0e50b8#r891746846
```suggestion
```